src/backends/interflop-bitmask/Makefile.am

```diff
@@ -1,9 +1,9 @@
 lib_LTLIBRARIES = libinterflop_bitmask.la
 libinterflop_bitmask_la_SOURCES = interflop_bitmask.c ../../common/logger.c ../../common/options.c
-libinterflop_bitmask_la_CFLAGS = -DBACKEND_HEADER="interflop_bitmask"
+libinterflop_bitmask_la_CFLAGS = -DBACKEND_HEADER="interflop_bitmask" -O3
```

Comment:
did you check O3 with various compiler? it can make a huge diff from one to another

Reply:
yes, icx, clang, and gcc.
And even test if o3 vs O0 and O2 for clang.
 I kind of remember long ago with the first version of quad, O3 was not working with a trick I brrowed from cadna to do up or down Stochastic rounding (1/2-1/2 probability), and we had this very same discussion for the other backends and conclude we do not have enough test to be confident at the time. 
I am more confident now about the test, but I rise the flag, maybe we should discuss it further with you and Pablo about potential impact and if this should be the default behavior or optional to use O3.

Reply:
I made some tests on NAS BT S and `clang` is faster than `gcc`.
| Backends version | Compiler  | IEEE |  mca int | mca |  
|------------|--------------|---------|--------|-----------|
| Original | `gcc-9`     |  1.22s         |   38.03s      | 58.94s  |
| Original | `clang-9`  |   1.25s        |    31.25s     | 54.11s |
| New | `gcc-9`     |   1.22s        |   30.35s      |    56.78s         |
| New | `clang-9`  |   1.26s        |    18.15s | 45.22s    |

Reply:
I was more worried about correctness, but it is very good to have some perf figure. It makes a huge difference, enough to spend the time looking for correctness if there is a glitch somewhere ;)

Reply:
> yes, icx, clang, and gcc. And even test if o3 vs O0 and O2 for clang. I kind of remember long ago with the first version of quad, O3 was not working with a trick I brrowed from cadna to do up or down Stochastic rounding (1/2-1/2 probability), and we had this very same discussion for the other backends and conclude we do not have enough test to be confident at the time. I am more confident now about the test, but I rise the flag, maybe we should discuss it further with you and Pablo about potential impact and if this should be the default behavior or optional to use O3.

For mca quad, `-O3` breaks the tests so it's definitely too aggressive, but maybe with specific flags (like `-ffp-model=strict` for `clang`) it could work but I did not test it. We could also add `-march=native` to trigger ISA but it will trouble docker images shipping, as the arch is not the same.